todoman/cli.py

```diff
@@ -353,6 +353,10 @@ def new(ctx, summary, list, todo_properties, read_description, interactive):
     if default_due:
         todo.due = todo.created_at + timedelta(hours=default_due)
 
+    default_priority = ctx.config['main']['default_priority']
+    if default_priority:
```

Comment:
This will be false if `default_priority=0`, so the todo will not have a priority set.

This will only work for non-zero priorities (thought according to the spec, unset == 0, that's not the intention there :stuck_out_tongue:  ).